doc/security/o_auth_configure.md

```diff
@@ -12,4 +12,4 @@ The following checklist provides the steps necessary to configure KAAZING Gatewa
 | 5        | Code your client to respond to the Gateway's authentication challenge.                                                | [Configure a Challenge Handler on the Client](p_auth_configure_challenge_handler.md)                                                                                                                        |
 | 6        | Configure the Gateway to specify the user roles that are authorized to perform operations for Gateway services.       | [Configure Authorization](p_authorization_configure.md)                                                                                                                                 |
 | 7        | Configure the Gateway to authorize or deny JMS operations performed by the client, using the JMSAuthorizationFactory. | [Secure the Connection from Each Client to the Gateway](p_client_jms_secure.md)                                                                                                          |
-| Optional | Inject bytes into a custom protocol or promote user credentials to the AMQP protocol.                                 |       [Implement Protocol Injection](o_aaa_inject.md) and [Implement User Identity Promotion](p_auth_protocol_injection.md)                                                                                                                                                                                   |
\ No newline at end of file
+| Optional | Inject bytes into a custom protocol or promote user credentials to the AMQP protocol.                                 |       [Implement Protocol Injection](p_auth_protocol_injection.md)                                                                                                                                                                                   |
```

Comment:
Update the link to point to the protocol injection topic in enterprise.gateway. It was accidentally merged into gateway (https://github.com/kaazing/gateway/issues/228).